docs/resources/resource_group_project.md

```diff
@@ -0,0 +1,46 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "oktapam_resource_group_project Resource - terraform-provider-oktapam"
+subcategory: ""
+description: |-
+  
+---
+
+# oktapam_resource_group_project (Resource)
+
+
+
+
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `name` (String) The human-readable name of the resource. Values are case-sensitive.
+- `resource_group` (String)
+
+### Optional
+
+- `account_discovery` (Boolean)
+- `create_server_users` (Boolean) If `true`, 'sftd' (ASA Server Agent) creates corresponding local (unix or windows) user accounts in the ASA Project's servers.
+- `forward_traffic` (Boolean) If `true`, all traffic in the ASA Project be forwarded through selected ASA Gateways.
+- `gateway_selector` (String) Assigns ASA Gateways with labels matching all selectors. At least one selector is required for traffic forwarding.
+- `next_unix_gid` (Number) The GID to use when creating a new ASA Server User. Default value starts at 63001.
```

Comment:
Recognize that we don't expose these values in current OktaPA view of servers. Can we exclude them from the provider (next UID/GID)?